models/tutorial.schema.js

```diff
@@ -0,0 +1,25 @@
+const mangoose = require('mongoose');
+
+const tutorialSchema = new mangoose.Schema({
+  id: {
+    type: String,
+    required: true,
+  },
+  published: {
+    type: Boolean,
+    required: true,
+  },
+  createdAt: {
```

Comment:
`createdAt` and `updatedAt` can be autogenerated using some options. Please follow this docs - https://masteringjs.io/tutorials/mongoose/timestamps